### PR TITLE
Update ironstrings.py to work in python3.

### PR DESCRIPTION
**Pull request by unk0unk0unk0**
Updated options for sorted function to work in python3.

### CODE REVIEW

File: python/flare/ironstrings/ironstrings.py [hunk 1]
```diff
@@ -161,15 +161,15 @@ def main():
...
+            for ss in sorted(f_ss, key=lambda s: s.written_at or 0):
                 print_string(ss.fva, ss.written_at, ss.offset, ss.s)
                 # IMPROVEMENT adjust stack frame member size in IDA view
 
             # print globalstrings found in this function
             f_gs = list(filter(lambda g: g.fva == fva, userData["globalstrings"]))
             cnt_found_ss += len(f_gs)
             for gs in sorted(f_gs, key=lambda g: g.written_at):
-                print_string(gs.fva, gs.written_at, gs.offset, gs.s)
+                print_string(gs.fva, gs.written_at, gs.offset, gs.s or 0)
```
Comment (mr-tz) [inline python/flare/ironstrings/ironstrings.py:172]: I have to inspect/test this in more detail, but from a first glance is this change supposed to be one line above?

Comment (unk0unk0unk0) [reply python/flare/ironstrings/ironstrings.py:172]: I'm very sorry!! It's my mistake.
No need to change line 172.
Instead, I should change line 171 to the following.
`for gs in sorted(f_gs, key=lambda g: g.written_at or 0):`

Comment (unk0unk0unk0) [reply python/flare/ironstrings/ironstrings.py]: I will close this pull request and submit new one to fix the above mistake.

**Comment by unk0unk0unk0** [comment]
I'll submit a new request to fix the mistake. Sorry.

**Pull request closed without merging**